internal/pkg/service/stream/api/service/sink.go

```diff
@@ -379,6 +379,22 @@ func (s *service) EnableSink(ctx context.Context, d dependencies.SinkRequestScop
 	return s.mapper.NewTaskResponse(t)
 }
 
+func (s *service) ListSinkVersions(ctx context.Context, scope dependencies.SinkRequestScope, payload *api.ListSinkVersionsPayload) (res *api.EntityVersions, err error) {
+	if err := s.sourceMustExists(ctx, scope.SourceKey()); err != nil {
```

Comment:
I believe you want sinkMustExists here and pass `scope.SinkKey()`.

Reply:
@jachym-tousek-keboola it is tested the same as the source

Reply:
sink existence check [c011c08](https://github.com/keboola/keboola-as-code/pull/2075/commits/c011c0878f26efba086a12d1c4dc9d9901c416a6)